gen/o365_life_cycle_manager_mu

```diff
@@ -177,16 +174,16 @@ sub processUsers {
 		}
 	#this is the first time user as occured in any processed group so we need to create a new full record
 	} else {
-		my $memberId = $memberAttributes{$A_MEMBER_ID};
-		my $preferredMail = $memberAttributes{$A_USER_PREFERRED_MAIL};
-		my $o365AlumniAccountEnabled = $memberAttributes{$A_USER_FAC_ALLUMNI_ACC_ENABLED};
-		my $o365GracePeriodExtension = $memberAttributes{$A_USER_FAC_GRACE_PERIOD_EXT};
-		my $eligibleForO365AlumniLicence = $memberAttributes{$A_USER_FAC_ELIGIBLE_FOR_LICENCE};
-		my $o365PrivilegedLicence = $memberAttributes{$A_USER_FAC_PRIVILEGED_LICENCE};
-		my $o365UserEmailAddresses = $memberAttributes{$A_USER_O365_USER_EMAILS_MU};
-		my $o365PrimaryEmailAddress = $memberAttributes{$A_USER_O365_PRIMARY_EMAIL_MU};
-		my $o365MailForward = $memberAttributes{$A_USER_FAC_O365_MAIL_FORWARD};
-		my $o365DeleteAccountRequest = $memberAttributes{$A_USER_FAC_DEL_ACC_REQUEST};
+		my $memberId = $data->getMemberAttributeValue( member => $memberId, attrName => $A_MEMBER_ID );
```

Comment:
You already know the member's id, because it is the initial parameter of this method now.

Reply:
Yeah, obviously you are right. Fixed.